sphinxcontrib/doxylink/doxylink.py

```diff
@@ -148,6 +148,8 @@ def parse_tag_file(doc: ET.ElementTree) -> dict:
 
     for member_symbol, arglist, kind, anchor_link in function_list:
         try:
+            if member_symbol[-10:] == 'operator()': # cleanup operator() because it confuses the normalization.
+                member_symbol = member_symbol[:-2] # just drop the trailing parens, should be non-ambiguous.
```

Comment:
the numbers `-10` and `-2` in there worry me a bit: is this dependent  on the number of spaces that are in the code? Does this work when the code has extra or less spaces? If not, i would suggest to use regex or pyparsing to get this done.

Reply:
It shouldn't depend on the code. That gets stripped before it's put in the tag file. The `-10` could be changed to `-len("operator()"`. The `2` is the same, it's `len("()")`. In this logic, those strings must match literally, it seems a bit overkill to use a regex to check for a literal string.

Reply:
Python isn't a main language for me, it's quite possible there are better ways to check for the string ending in literally "operator()".

Reply:
You can use [`str.endswith()`](https://docs.python.org/3/library/stdtypes.html#str.endswith) to check whether a string has a certain suffix in Python.